webui/package.json

```diff
@@ -44,7 +44,5 @@
       "last 1 safari version"
     ]
   },
-  "devDependencies": {
-    "typescript": "^4.0.5"
-  }
+  "devDependencies": {}
```

Comment:
```suggestion
```
![Y U no luv JSON??!?](https://user-images.githubusercontent.com/7802932/102894076-35d0d500-446b-11eb-9b91-02ab44eac430.png)

Reply:
npm uninstall did it